src/core/fem/src/condition/4C_fem_condition_locsys.cpp

```diff
@@ -100,6 +100,19 @@ void Core::Conditions::LocsysManager::update(const double time,
   geo_hierarchy.emplace_back(Core::Conditions::LineLocsys);
   geo_hierarchy.emplace_back(Core::Conditions::PointLocsys);
 
+  // Ensure locsystoggle_ is on the node *row* (owned/unique) map
+  const Core::LinAlg::Map& node_row_map = *discret_.node_row_map();
+
+  if (!locsystoggle_ || !locsystoggle_->get_map().same_as(node_row_map))
+  {
+    // (Re)create on the correct map
+    locsystoggle_ = Core::LinAlg::create_vector(node_row_map, /*zero*/ true);
+  }
+
+  // Optional: sanity print once
+  FOUR_C_ASSERT(
+      locsystoggle_->get_map().same_as(node_row_map), "locsystoggle_ must use node row map");
```

Comment:
Is this still relevant? Doesn't the content of the `if` make sure, that this error will never be triggered?